spec/Overview.html

```diff
@@ -6882,9 +6837,7 @@ <h4>Operations</h4>
                         </li>
                         <li>
                           <p>
-                            Let |result| be a new empty {{ArrayBuffer}} associated with the
-                            [= relevant global object =]
-                            of `this` [[HTML]].
+                            Let |result| be a new empty {{ArrayBuffer}}.
```

Comment:
This one seems wrong because this creates a non-growable ArrayBuffer whose length is 0, but the steps which follow “append bytes” to it. Presumably the intent is to append to `result.[[ArrayBufferData]]`, but even so, that isn’t possible — that [Data Block](https://tc39.es/ecma262/#sec-data-blocks)’s size is fixed.

I’m not 100% sure this would be the right way to go but my best guess is that this step should really be creating a new empty [byte sequence](https://infra.spec.whatwg.org/#byte-sequences) for the accumulation steps. The ArrayBuffer would be created afterwards, the byte sequence now being the data argument for “create”.